lib/AppInfo/Application.php

```diff
@@ -79,6 +80,7 @@ public function register()
                     exit();
                 });
             }
+	    Util::addStyle($this->appName, 'server'); 
```

Comment:
Is this indent correct? Also this needs to be configurable.

Reply:
The indent is a typo—sorry for missing the target.

Unfortunately more urgent priorities interfere, so I won't be able to add a UI to this — very sorry… Hopefully you or some other contributor might pick it up from here.

Reply:
I'll pick it up don't worry. No UI, btw. The whole plugin's config is deliberately done through the global config.

Reply:
I actually did it, no worries. Pls allow me a few minutes to update.

Reply:
Continued as PR #60.